tempesta_fw/Makefile

```diff
@@ -42,9 +42,8 @@ tempesta_fw-objs = \
 	main.o \
 	pool.o \
 	sched.o \
-	server.o \
+	server_be.o \
```

Comment:
Is there any reason for not calling it something like ```server_backend```?
Also, I am looking for name consistency. We used to have ```sock_backend``` and ```sock_frontend```. Now we have ```server_be`` and ```sock_frontend```. This is quite confusing.